apps/glusterfs/node_entry.go

```diff
@@ -68,6 +69,49 @@ func (n *NodeEntry) registerStorageKey(host string) string {
 	return "STORAGE" + host
 }
 
+// Verify gluster process in the node and return the manage hostname of a node in the cluster
+func GetNode(db *bolt.DB, e executors.Executor, clusterId string) (string, error) {
+	godbc.Require(clusterId != "")
+	var cluster *ClusterEntry
+	var node *NodeEntry
+	var err error
+	err = db.View(func(tx *bolt.Tx) error {
+		var err error
+		cluster, err = NewClusterEntryFromId(tx, clusterId)
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
if you don't take any special action upon err != nil, you can just return err here?